core/src/main/scala/com/pingcap/tispark/v2/TiDBTable.scala

```diff
@@ -128,21 +130,31 @@ case class TiDBTable(
 
   override def newWriteBuilder(info: LogicalWriteInfo): WriteBuilder = {
     var scalaMap = info.options().asScala.toMap
-    // TODO https://github.com/pingcap/tispark/issues/2269 we need to move TiDB dependencies which will block insert SQL.
-    // if we don't support it before release, insert SQL should throw exception in catalyst
-    if (scalaMap.isEmpty) {
-      throw new TiBatchWriteException("tidbOption is neccessary.")
+    // For update statics
+    val enableUpdateStatics =
+      sqlContext.getConf(TiDBOptions.TIDB_ENABLE_UPDATE_TABLE_STATISTICS, "false").toBoolean
+    if (enableUpdateStatics) {
```

Comment:
Just judge if the tidb config exists. if so, add them to scalaMap.
Judge by enableUpdateStatics is not a good idea. what if we need tidb config in another situation？